cmd/engine/.dagger/build/builder.go

```diff
@@ -139,15 +139,15 @@ func (build *Builder) Engine(ctx context.Context) (*dagger.Container, error) {
 				Arch: build.platformSpec.Architecture,
 			}).
 			Container().
-			WithExec([]string{"sh", "-c", `
-				set -e
-				ln -s /sbin/iptables-legacy /usr/sbin/iptables
-				ln -s /sbin/iptables-legacy-save /usr/sbin/iptables-save
-				ln -s /sbin/iptables-legacy-restore /usr/sbin/iptables-restore
-				ln -s /sbin/ip6tables-legacy /usr/sbin/ip6tables
-				ln -s /sbin/ip6tables-legacy-save /usr/sbin/ip6tables-save
-				ln -s /sbin/ip6tables-legacy-restore /usr/sbin/ip6tables-restore
-			`})
+			WithExec([]string{"sh", "-c", strings.Join([]string{
+				"mkdir -p /usr/local/sbin",
+				"ln -s /usr/sbin/iptables-legacy /usr/local/sbin/iptables",
```

Comment:
Alpine 3.22 moves a lot of binaries into `/usr/sbin`, so we make our links in `/usr/local/bin` (which has more path priority, and apk should never touch).

See https://github.com/dagger/dagger/pull/7670 for context, fyi @marcosnils